PWGHF/HFC/DataModel/CorrelationTables.h

```diff
@@ -200,6 +205,64 @@ DECLARE_SOA_TABLE(DplusHadronRecoInfo, "AOD", "DPLUSHRECOINFO", //! D+-Hadrons p
                   aod::hf_correlation_dplus_hadron::MD,
                   aod::hf_correlation_dplus_hadron::SignalStatus);
 
+// definition of columns and tables for Dstar-Hadron correlation pair
+using Dstars = HfCandDstar;
+using Dstar = Dstars::iterator;
+namespace hf_correlation_dstar_hadron
+{
+DECLARE_SOA_INDEX_COLUMN(Collision, collision);
+// Dstar candidate properties
+DECLARE_SOA_INDEX_COLUMN(Dstar, dstar);
+DECLARE_SOA_COLUMN(PhiDstar, phiDstar, float);
+DECLARE_SOA_COLUMN(EtaDstar, etaDstar, float);
+DECLARE_SOA_COLUMN(PtDstar, ptDstar, float);
+DECLARE_SOA_COLUMN(MDstar, mDstar, float);
+// DECLARE_SOA_COLUMN(IsPrompt,isPrompt,bool); // although this also defined in (HfCandDstarMcRec HfCandDstarMcRec) tables
+// DECLARE_SOA_COLUMN(MatchingStatus, matchingStatus, bool); // although this also defined in (HfCandDstarMcRec HfCandDstarMcRec) tables
+// Track properties
+DECLARE_SOA_INDEX_COLUMN(Track, track);
+DECLARE_SOA_COLUMN(PhiTrack, phiTrack, float);
+DECLARE_SOA_COLUMN(EtaTrack, etaTrack, float);
+DECLARE_SOA_COLUMN(PtTrack, ptTrack, float);
+// common
+DECLARE_SOA_COLUMN(TimeStamp, timeStamp, int64_t);
+DECLARE_SOA_COLUMN(PoolBin, poolBin, int);
+// Dynamic columns
+DECLARE_SOA_DYNAMIC_COLUMN(DeltaEta, deltaEta, [](float etaTrack, float etaCandidate) -> float { return (etaTrack - etaCandidate); });
+DECLARE_SOA_DYNAMIC_COLUMN(DeltaPhi, deltaPhi, [](float phiCandidate, float phiTrack) -> float { return RecoDecay::constrainAngle(phiTrack, phiCandidate); });
+} // namespace hf_correlation_dstar_hadron
+
+DECLARE_SOA_TABLE(DstarHadronPair, "AOD", "DSTRHPAIR", // D* Hadrons pairs Informations
+                  hf_correlation_dstar_hadron::CollisionId,
+                  // D* only properties
+                  hf_correlation_dstar_hadron::DstarId,
+                  hf_correlation_dstar_hadron::PhiDstar,
+                  hf_correlation_dstar_hadron::EtaDstar,
+                  hf_correlation_dstar_hadron::PtDstar,
+                  hf_correlation_dstar_hadron::MDstar,
+                  // Track only properties
+                  hf_correlation_dstar_hadron::TrackId,
+                  hf_correlation_dstar_hadron::PhiTrack,
+                  hf_correlation_dstar_hadron::EtaTrack,
+                  hf_correlation_dstar_hadron::PtTrack,
+                  // common
+                  hf_correlation_dstar_hadron::TimeStamp,
+                  hf_correlation_dstar_hadron::PoolBin,
+                  // common Dynamic
+                  hf_correlation_dstar_hadron::DeltaPhi<hf_correlation_dstar_hadron::PhiDstar, hf_correlation_dstar_hadron::PhiTrack>,
+                  hf_correlation_dstar_hadron::DeltaEta<hf_correlation_dstar_hadron::EtaDstar, hf_correlation_dstar_hadron::EtaTrack>);
+
+// Table for mc matched info
+using RecoMatchingInfoDstars = HfCandDstarMcRec;
+using RecoMatchingInfoDstar = RecoMatchingInfoDstars::iterator;
```

Comment:
Why can you not use the original name?

Reply:
same reason as given for the above case 
using Dstars = HfCandDstar;
using Dstar = Dstars::iterator;

Reply:
Same reply as above